_data/y2019/sessions.yaml

```diff
@@ -3,32 +3,31 @@
   start_time: 13:00
   end_time: 13:20
   duration: 00:20
-  title: Let me talk about my vimrc to grow your vimrc
+  title: Your Vim Is Only for You
```

Comment:
This capitalization is right?
please teach me if you know.

Reply:
ping @ujihisa

Reply:
maybe "Your Vim is only for Vim"

Reply:
```suggestion
  title: Your Vim is Only for You
```

Reply:
:special_thanks: :+1: